app/src/main/java/com/sun/chat_04/ui/signup/SignUpActivity.kt

```diff
@@ -0,0 +1,148 @@
+package com.sun.chat_04.ui.signup
+
+import android.Manifest
+import android.annotation.SuppressLint
+import android.app.DatePickerDialog
+import android.content.pm.PackageManager
+import android.os.Bundle
+import android.support.v4.app.ActivityCompat
+import android.support.v4.content.ContextCompat
+import android.support.v7.app.AppCompatActivity
+import android.view.View
+import android.view.View.OnClickListener
+import android.widget.DatePicker
+import android.widget.RadioButton
+import com.jpardogo.android.googleprogressbar.library.ChromeFloatingCirclesDrawable
+import com.sun.chat_04.R.color
+import com.sun.chat_04.R.id
+import com.sun.chat_04.R.layout
+import kotlinx.android.synthetic.main.sign_up_screen.button_signUp
+import kotlinx.android.synthetic.main.sign_up_screen.edit_confirm_password
+import kotlinx.android.synthetic.main.sign_up_screen.edit_email
+import kotlinx.android.synthetic.main.sign_up_screen.edit_fullname
+import kotlinx.android.synthetic.main.sign_up_screen.edit_password
+import kotlinx.android.synthetic.main.sign_up_screen.progress
+import kotlinx.android.synthetic.main.sign_up_screen.radio_groups
+import kotlinx.android.synthetic.main.sign_up_screen.text_birthday
+import kotlinx.android.synthetic.main.sign_up_screen.toolbar
+import java.text.SimpleDateFormat
+import java.util.Calendar
+
+class SignUpActivity : AppCompatActivity(), OnClickListener {
+    private var mBirth: String = ""
```

Comment:
Kotlin có trình tự động suy luận kiểu mạnh mẽ, e k cần xác định rõ type
```suggestion
    private var mBirth = ""
```
lưu ý xem có nên tạo các biến này là instancr variable k nhé